front/components/V2/SearchHistory.tsx

```diff
@@ -0,0 +1,70 @@
+import { View } from 'react-native';
+import { Text } from 'native-base';
+import { useQuery } from '../../Queries';
+import API from '../../API';
+import { translate } from '../../i18n/i18n';
+import { LoadingView } from '../Loading';
+
+type historyRowProps = {
+	type: string;
+	query: string;
+	timestamp: Date;
+};
+
+const HistoryRowComponent = (props: historyRowProps) => {
+	return (
+		<View
+			style={{
+				borderTopWidth: 1,
+				borderTopColor: '#9E9E9E',
```

Comment:
i think you should use the theme to get colors instead of hard coding them (it won't work on both dark and light theme otherwise)

Reply:
same for the 'gray' bellow